src/Handler/CardUpdateHandler.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Handler;
+
+use App\Entity\Card;
+use App\Exception\NotUniqueCardnameException;
+use App\Request\EditCard;
+use App\Validator\CardDataValidator;
+use Doctrine\ORM\EntityManagerInterface;
+use Exception;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+class CardUpdateHandler
+{
+    public function __construct(
+        private EditCard $editCard,
```

Comment:
In common constructor is use to inject dependency to class. Dependency is other part of code like object which handle logic, data with configuration and other thing that could help to process something. But `private EditCard $editCard` is not that. `EditCard` is a command, and that command should be passed into this class as argument in `handle` method. If we do that ten we could use that class in other places in our appliaction. And create command by `new`. But now we couldn't because its required ParamConverter or instantiation this handler by `new` which is wrong because we cannot use our DI configuration.